flink-runtime/src/test/java/org/apache/flink/runtime/highavailability/AbstractHaServicesTest.java

```diff
@@ -97,7 +98,8 @@ void testCloseAndCleanupAllDataDoesNotDeleteBlobsIfCleaningUpHADataFails() throw
                         },
                         ignored -> {});
 
-        assertThatThrownBy(haServices::closeAndCleanupAllData).isInstanceOf(FlinkException.class);
+        assertThatThrownBy(haServices::cleanupAllData).isInstanceOf(FlinkException.class);
+        haServices.close();
```

Comment:
```suggestion
        assertThatThrownBy(() -> haServices.closeWithOptionalCleanup(true)).isInstanceOf(FlinkException.class);
```
This improves the test coverage (i.e. it would check the newly added default implementation).